.github/workflows/python-tests.yml

```diff
@@ -87,4 +52,12 @@ jobs:
     - name: Install Tox
       run: python -m pip install tox
     - name: Run Tox
-      run: tox ${{ matrix.toxargs }} -v -e ${{ matrix.toxenv }}
+      env:
+        PY: ${{ matrix.python-version }}
+        PT: ${{ matrix.pytest-version }}
+      run: |
+        # crete the token as "pyXY-test-pytestXY" from the matrix with drop dots
```

Comment:
The comment has typo and I don't know what "drop dots" mean.

Reply:
the base python notation is `3.8` with dot but for env you use just numbers `38`

Reply:
If I see correctly this matrix update will have way more jobs, as we haven't tested all old versions with all OSes, etc, and have already received criticism that there are way too many jobs for the versions in the middle.